### ISSUE TITLE
This algorithm should be completely rethinked #2

### ISSUE THREAD

**Issue by Sires0** [archfan7411/clearplayers#3]
First, I fell that it's more convenient opening another issue for that, since that editing the first would be confusing.
Not every player without certain items is just useless, MANY times this is because the player was clearing his inventory for doing something else(like selling cobble in my case) and couldn't continue in this day, this algorithm would destroy many players that are not inactive. I have a better sollution for the algorithm that would be easier too.
My solution involves a lua mod and the python program, a lua mod that checks if the player have been inactive and didn't do much progress(the mod I could do) and if the player is, it sets a player attribute like 
`inactive = "true"`
that could be easily detected.
The "part two of the part two" is that read function does not take the line number to read as param, it takes how many bytes to read, so you could use readlines or something instead, that with my solution would be even easier

**Comment by archfan7411** [comment]
Brilliant! This is much better than my solution. Although maybe I should keep developing an option for lower quality script-only work (for those who do not wish to install the mod), this would work far better for the server.

**Comment by archfan7411** [comment]
Of course, it should be noted that the only change a player would experience upon having their account deleted would be loss of inventory and being reset to the spawn; all their locked things would still belong to their player name.

**Comment by archfan7411** [comment]
Whoops, accidentally closed, sorry. Reopened.

**Comment by Sires0** [comment]
![suususu](https://user-images.githubusercontent.com/18670349/33717761-4851ed62-db42-11e7-8274-7ee4e3d475c8.png)

Hey what about me, the one who like keeping his very important stuff(obdy and acacia wood) in his 4 large bags?

Many ppl in the server have bags and MOST of the time they use it when they want to clean the inventory, in my personal case I have 4 almost full large bags(going a bit more offtopic, I saw someone buy my bag that I sell and MichaelEh makes bags even cheaper than me, so many players may have bags and other servers this may happen too)! 

All the inventories attached to the player are saved in the player file, so imagine the player was mining cobble hard and bought 4 large bags from SiresEh(what a creative name) but then had to quit? 

The new 96 empty spaces will be more than enough to delete him and his stuff, even if he has a full main inventory.

So the thing is, don't calculate by how many empty spaces the player has, calculate how many full spaces the player has, IMHO 3 full spaces is enough, many players join at night and start doing stuff, mine some cobble and get clay and then quit cuz it's night.

**Comment by Sires0** [comment]
Also, this is a small project, I don't think creating a new label for each issue is the better lel

**Comment by archfan7411** [comment]
> Also, this is a small project, I don't think creating a new label for each issue is the better lel

Sorry, new to GitHub...

> So the thing is, don't calculate by how many empty spaces the player has, calculate how many full spaces the player has, IMHO 3 full spaces is enough, many players join at night and start doing stuff, mine some cobble and get clay and then quit cuz it's night.

Ok, I'll try to put that in the script, but with real life, working on another to-be-revealed mod for the server, and updating the shuttle mod, it might be a day or two before I make the changes.
